spec/rubocop/cop/layout/else_alignment_spec.rb

```diff
@@ -535,4 +535,36 @@ def my_func
       RUBY
     end
   end
+
+  context 'ensure/rescue/else in Block Argument' do
+    let(:ruby_version) { 2.5 }
```

Comment:
You can use `ruby25` shared example. https://github.com/bbatsov/rubocop/blob/ef9dafce377aea7681b1db8591e130d076d73017/spec/rubocop/cop/layout/rescue_ensure_alignment_spec.rb#L101